### ISSUE TITLE
Social-value motivations > Platform-centric demos

### ISSUE THREAD

**Issue by benhylau** [dweb-camp-2019/projects#4]
I would personally like to see more projects that start from _social-value motivations_ over _platform-centric demos_. For example...

**Platform-centric:**

>Deploy this application across the network to provide distributed storage.

**Social-value motivated:**

>Using distributed storage systems, install nodes on the local network and map the physical and virtual infrastructures, followed by a discussion session about community governance models that ensure local ownership and digital sovereignty.

I think the latter example provides more opportunity for bidirectional learning across disciplines and specialities.

**Comment by alexalekseyenko** [comment]
To that end, do you think it would be useful to define some capabilities that we'd likely want to have as a community? For example:
 - Chat/broadcast to transmit messages to those who could benefit from them
 - Schedule to let organizers communicate the days' activities
 - Map to display points of interest

 Or to list some topics we'd like to explore? Such as:
 - Community governance models that ensure local ownership and digital sovereignty
 - Considerations for designing mesh network infrastructure
 - Designing applications for limited connectivity environments

 If you think it would be useful to have such lists, would this be the best place to construct them?

**Comment by amaliejames** [comment]
I agree that a community board for those kinds of lists would be helpful! For example, we intend to use camp as a real world test to see what happens if network falls out of sync with the internet in a low connectivity environment. Would be awesome to find folks who are curious about similar questions to connect with!

**Comment by benhylau** [comment]
@alexalekseyenko oops just saw your message, sorry I missed it before.

## Capabilities

>Chat/broadcast to transmit messages to those who could benefit from them

For example, GUN https://github.com/dweb-camp-2019/applications/issues/2 SSB https://github.com/dweb-camp-2019/applications/issues/6 Cabal https://github.com/dweb-camp-2019/applications/issues/7 are all related to the same thread.

## Topics

I think https://github.com/dweb-camp-2019/applications/issues/12 is one of these topics

---

My thinking is to use the Issues board on this repository as the place to advertise things and store discussion summaries (but perhaps there are better tools for a "community board"?) and use [#dweb-camp-2019-apps:matrix.org](https://riot.im/app/#/room/#dweb-camp-2019-apps:matrix.org) as a place for synchronous discussions?

How about posting chat handles in Issues like...

>If you are interested to discuss more, ping me on `#dweb-camp-2019-apps:matrix.org`, I am `@benhylau:tomesh.net`!

I can make an Issue template to help with this, or create new Issues tags like `Capabilities` and `Topics`?

Thoughts? @alexalekseyenko @amaliejames @amberinitiative @ArtsEd4All

**Comment by alexalekseyenko** [comment]
No problem. Thank you for following up.

I did notice that there were multiple issues regarding chat, and I thought that might be an indication that high level planning would be useful. Namely, there are probably some basic things we can anticipate people needing, like chat, or an up-to-date schedule, and it would be good to have at least one solution working and tested well in advance.

GitHub Issues are great for tracking individual work items, but I'm not sure that they provide a good high level overview of the situation (in that way that lengthy lists of things usually don't). To that end, I think it could be helpful to synthesize that information and to add it to the README.

What that might look like:
## Applications
### Chat
SSB at a minimum `link to relevant issue`
Other options in the works `links` `to` `other` `issues`
### Schedule
GitHub README at a minimum `link to page`
Other options in the works `links` `if` `there` `are` `any`
### Map
Does this even make sense? `link to discussion issue`

**Comment by benhylau** [comment]
@alexalekseyenko I sent a PR using new labels to group. That is perhaps not as detailed as your example above (e.g. doesn't really summarize the current status of each category). Do you think we should be more explicit?

Also invited you as collaborator to this repo if you'd like to be more involved with the curation :)

**Comment by benhylau** [comment]
@amaliejames would a posting of something like _Explore sync behaviour of distributed protocols under poor connectivity environments_ to the [`Discussion Topic`](https://github.com/dweb-camp-2019/projects/issues?q=is%3Aissue+is%3Aopen+label%3A%22Discussion+Topic%22) label be sufficient to attract people to that interest?

**Comment by alexalekseyenko** [comment]
@benhylau Thank you for adding this to the page. It feels like a more gentle introduction to what's happening.
